keras_cv/layers/object_detection/multi_class_non_max_suppression_test.py

```diff
@@ -12,30 +12,62 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import unittest
+
 import tensorflow as tf
 
 from keras_cv import layers as cv_layers
 
 
+def decode_predictions_output_shapes():
+    classes = 10
+    predictions_shape = (8, 98208, 4 + classes)
+
+    predictions = tf.random.uniform(
+        shape=predictions_shape, minval=0.0, maxval=1.0, dtype=tf.float32
+    )
+    box_pred = predictions[..., :4]
+    confidence_pred = predictions[..., 4:]
+
+    layer = cv_layers.MultiClassNonMaxSuppression(
+        bounding_box_format="xyxy",
+        from_logits=True,
+        max_detections=100,
+    )
+
+    result = layer(box_prediction=box_pred, confidence_prediction=confidence_pred)
+    return result
```

Comment:
The only difference now is that we expect to return the output from the compiled function. This is triggered the failure with the MLIR bridge.

Reply:
https://www.tensorflow.org/xla/known_issues?hl=en#random_number_generation_ignores_tf_seed